src/web/js/trove/world-lib.js

```diff
@@ -750,10 +759,20 @@
                 k2();
               } else {
                 if (extras.closeWhenStop) {
-                  if (extras.closeBigBangWindow) {
-                    extras.closeBigBangWindow();
+                  if (lif) {
+                    runtime.ffi.throwMessageException('Cannot have both last-image and close-when-stop(true)');
```

Comment:
Two things – if there's a dynamic error running a world, it should probably still shut down the world.  Also, why even _start_ the process running if there's an error?  Can't we just warn up front that this combination is disallowed?